src/_P050_TCS34725.ino

```diff
@@ -83,27 +84,40 @@ boolean Plugin_050(byte function, struct EventStruct *event, String& string)
 
       byte   choiceMode2 = PCONFIG(1);
       String optionsMode2[4];
-      optionsMode2[0] = F("TCS34725_GAIN_1X");
-      optionsMode2[1] = F("TCS34725_GAIN_4X");
-      optionsMode2[2] = F("TCS34725_GAIN_16X");
-      optionsMode2[3] = F("TCS34725_GAIN_60X");
+      optionsMode2[0] = F("1x");
+      optionsMode2[1] = F("4x");
+      optionsMode2[2] = F("16x");
+      optionsMode2[3] = F("60x");
       int optionValuesMode2[4];
       optionValuesMode2[0] = TCS34725_GAIN_1X;
       optionValuesMode2[1] = TCS34725_GAIN_4X;
       optionValuesMode2[2] = TCS34725_GAIN_16X;
       optionValuesMode2[3] = TCS34725_GAIN_60X;
       addFormSelector(F("Gain"), F("p050_gain"), 4, optionsMode2, optionValuesMode2, choiceMode2);
 
+      addFormSubHeader(F("Output settings"));
+
+      addFormCheckBox(F("Output calibrated RGB values"), F("p050_calibratedrgb"), PCONFIG(2) == 1, false);
+
+      String optionsOutput[4];
+      optionsOutput[0] = F("Color Temperature");
+      optionsOutput[1] = F("Color Temperature (DN40)");
+      optionsOutput[2] = F("Lux");
+      optionsOutput[3] = F("Clear");
+      int optionValuesOutput[4] = { 0, 1, 2, 3};
+      addFormSelector(F("Output at Values #4"), F("p050_output4"), 4, optionsOutput, optionValuesOutput, PCONFIG(3));
+      addFormNote(F("Optionally adjust Values #4 name accordingly."));
```

Comment:
Please place all these options + call to `addFormSelector` in their own scope (`{}`) so they can be destructed as soon as they are sent to the web client.

Reply:
Yes, will fix.